exonum-java-binding-qa-service/src/main/java/com/exonum/binding/qaservice/transactions/BiDirectionTransactionConverter.java

```diff
@@ -16,37 +16,33 @@
 
 package com.exonum.binding.qaservice.transactions;
 
-import com.exonum.binding.common.message.BinaryMessage;
-import com.exonum.binding.common.message.Message;
 import com.exonum.binding.qaservice.PromoteToCore;
+import com.exonum.binding.transaction.RawTransaction;
 import com.exonum.binding.transaction.Transaction;
 
 /**
- * A converter between executable transaction & its binary message.
+ * A converter between executable transaction and Exonum binary raw transaction data.
  *
  * @param <TransactionT> a type of transaction. You can have a single
- *     {@link TransactionMessageConverter} that works with any transactions of this service
+ *     {@link BiDirectionTransactionConverter} that works with any transactions of this service
  *     or a converter for each transaction type.
  */
 @PromoteToCore("… with first-class support of transactions that don’t need a message to operate."
```

Comment:
I'd remove this comment, as I don't see a place for such an interface in the core.